### PR TITLE
Improvements in Axp192

### PR DESCRIPTION
**Pull request by josesimoes**
## Description
- Add several constants to improve code.
- Fix typo in method names.
- Fix Intellisense comments.
- Improve nuspec.
- Migrate solution to VS2022.

## Types of changes

- [x] Improvement (non-breaking change that improves a feature, code or algorithm)
- [ ] Bug fix (non-breaking change which fixes an issue with code or algorithm)
- [ ] New feature (non-breaking change which adds functionality to code)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Config and build (change in the configuration and build system, has no impact on code or features)
- [ ] Dependencies (update dependencies and changes associated, has no impact on code or features)
- [ ] Unit Tests (add new Unit Test(s) or improved existing one(s), has no impact on code or features)
- [ ] Documentation (changes or updates in the documentation, has no impact on code or features)

## Checklist:

- [x] My code follows the code style of this project (only if there are changes in source code).
- [x] My changes require an update to the documentation (there are changes that require the docs website to be updated).
- [ ] I have updated the documentation accordingly (the changes require an update on the docs in this repo).
- [x] I have read the [CONTRIBUTING](https://github.com/nanoframework/.github/blob/master/CONTRIBUTING.md) document.
- [ ] I have tested everything locally and all new and existing tests passed (only if there are changes in source code).
- [ ] I have added new tests to cover my changes.

### CODE REVIEW

File: devices/Axp192/Axp192.cs [hunk 1]
```diff
@@ -67,7 +71,7 @@ public ElectricPotential LDO3OutputVoltage
         /// Gets or sets DC-DC2 voltage.
         /// </summary>
         /// <remarks>Range is from 0.7 to 2.275V, steps of 25 mV</remarks>
-        public ElectricPotential DcDc2Volvate
+        public ElectricPotential DcDc2Volvage
```
Comment (Ellerbach) [inline devices/Axp192/Axp192.cs:74]:
```suggestion
        public ElectricPotential DcDc2Voltage
```

File: devices/Axp192/Axp192.cs [hunk 2]
```diff
@@ -87,7 +91,7 @@ public ElectricPotential DcDc2Volvate
         /// Gets or sets DC-DC1 voltage.
         /// </summary>
         /// <remarks>Range is from 0.7 to 3.5V, steps of 25 mV</remarks>
-        public ElectricPotential DcDc1Volvate
+        public ElectricPotential DcDc1Volvage
```
Comment (Ellerbach) [inline devices/Axp192/Axp192.cs:94]:
```suggestion
        public ElectricPotential DcDc1Voltage
```

File: devices/Axp192/Axp192.cs [hunk 3]
```diff
@@ -106,7 +110,7 @@ public ElectricPotential DcDc1Volvate
         /// Gets or sets DC-DC3 voltage.
         /// </summary>
         /// <remarks>Range is from 0.7 to 2.275V, steps of 25 mV</remarks>
-        public ElectricPotential DcDc3Volvate
+        public ElectricPotential DcDc3Volvage
```
Comment (Ellerbach) [inline devices/Axp192/Axp192.cs:113]:
```suggestion
        public ElectricPotential DcDc3Voltage
```

File: devices/Axp192/Axp192.cs (hunk 1 above)
Comment (josesimoes) [reply devices/Axp192/Axp192.cs:74]: WOW!!! A typo in a typo! 😅

**Comment by josesimoes** [comment]
> Looks good to me. Once merged, just make sure to update all the needed M5Stack! Axp192 is used a lot there

Sure thing! I stumbled on this when working on the BSP for the Tough.

**Pull request merged**